kotlin/query-with-cql-filters/README.metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "category": "Layers",
```

Comment:
do Android have categories? check with @TADraeseke what best practice is here. The category should be OGC.